src/Compiler/Backend/GLSL/GLSLConverter.cpp

```diff
@@ -813,6 +813,9 @@ void GLSLConverter::ConvertIntrinsicCall(CallExpr* ast)
         case Intrinsic::Texture_SampleLevel_5:
             ConvertIntrinsicCallTextureSampleLevel(ast);
             break;
+        case Intrinsic::Image_Store:
```

Comment:
Ok, so the `imageStore` conversion might need an additional conversion afterwards?!